libs/ui/breadcrumb/helm/src/lib/breadcrumb-separator.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, computed, input } from '@angular/core';
+import { HlmIconComponent, provideIcons } from '@spartan-ng/ui-icon-helm';
+import { lucideChevronRight } from '@ng-icons/lucide';
+import { hlm } from '@spartan-ng/ui-core';
+
+@Component({
+   
+  selector: 'hlm-breadcrumb-separator',
+  standalone: true,
+  imports: [HlmIconComponent],
+  providers: [provideIcons({ lucideChevronRight })],
+  host: {
+    '[class]': '_computedClass()',
+  },
+  template: `
+    <div #ref>
```

Comment:
Empty `div` is rendered when no content projection is added. Maybe there is a way to avoid the empty div.

Reply:
I considered different approaches when I first implemented this component, but I didn't find a better solution for that case.
IMHO, an empty `div` in case there is no content for this component is acceptable